inference-engine/tests/functional/plugin/myriad/subgraph_tests/dsr_strided_slice.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "dsr_tests_common.hpp"
+
+#include <functional_test_utils/layer_test_utils.hpp>
+#include <ngraph_functions/builders.hpp>
+#include <vpu/ngraph/operations/dynamic_shape_resolver.hpp>
+
+namespace {
+
+using DataType = ngraph::element::Type_t;
```

Comment:
Such `defines` are already defined in `LayerTestsUtils::vpu` namespace

Reply:
Thanks, removed.